packages/lpa-questionnaire-web-app/src/views/development-plan.njk

```diff
@@ -46,6 +46,7 @@
             label: {
               text: "Tell us about any substantive changes in the progress of the emerging plan, and their relevance to this appeal (if the plan may not be adopted before the Inspector issues their decision)."
             },
+            id: "plan-changes-text",
             idPrefix: "plan-changes-text",
```

Comment:
I don't think this line is needed. Best to remove. Avoids issues if for some reason it gets used in future

Reply:
Done.